library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -160,6 +161,7 @@ class Client() {
         privateKeyBundleV1: PrivateKeyBundleV1,
         apiClient: ApiClient,
         libXMTPClient: FfiXmtpClient? = null,
+        dbPath: String = ""
```

Comment:
Instead of having to store the dbPath like this is it something the `FfiXmtpClient` could expose?

Reply:
We could keep track of that and expose it. We have the info